ocaml/xapi/cert_refresh.ml

```diff
@@ -50,6 +50,12 @@ let new_host_cert ~dbg ~path : X509.Certificate.t =
   let ips = [ip] in
   Gencertlib.Selfcert.host ~name ~dns_names ~ips path
 
+let reload_stunnel () =
+  let systemctl cmd =
+    Helpers.call_script !Xapi_globs.systemctl [cmd; "stunnel@xapi"] |> ignore
+  in
+  systemctl "reload-or-restart"
+
```

Comment:
For consistency you could put this function in `Helpers.Stunnel`

We'll probably want to use it here too https://github.com/xapi-project/xen-api/blob/99d1f2d9c34265ebf0f78650960f3a4b5fb8fc87/ocaml/xapi/cert_distrib.ml#L229 (though that is out of scope for this PR)

Reply:
Done